resync/api/monitoring_dashboard.py

```
@@ -244,70 +199,66 @@ async def _check_alerts_redis(self, sample: MetricSample) -> None:
+                    if existing is None:
+                        await redis.set(REDIS_KEY_START_TIME, str(now))
                 finally:
                     await lock.release()
-            
+
             raw = await redis.get(REDIS_KEY_START_TIME)
             if raw is None:
                 logger.warning("Não foi possível determinar o tempo de início global do Redis.")
                 return 0.0
             return now - float(raw)
         except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)